tutorials/18_training_interface.ipynb

```diff
@@ -303,63 +284,193 @@
-      "text/plain": [
-       "<Figure size 300x300 with 4 Axes>"
-      ]
-     },
-     "metadata": {},
-     "output_type": "display_data"
-    }
-   ],
+   "outputs": [],
    "source": [
     "_ = pairplot(samples, limits=[[-3, 3], [-3, 3]], figsize=(3, 3), upper=\"contour\")"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "5a5b9e60-f504-4837-9ce8-c5af63dd03bd",
+   "id": "d3c92a48-d6ff-4e06-a842-47c0e07de217",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "### Custom Data Loaders\n",
+    "\n",
+    "One helpful advantage of having access to the training loop is, that you can now use your own DataLoaders during training of the density estimator. In this fashion, larger datasets can be used as input to `sbi` where `x` is potentially an image or something else. While this will require [embedding the input data](https://sbi-dev.github.io/sbi/latest/tutorials/04_embedding_networks/), a more fine grained control over loading the data is possible and allows to manage the memory requirement during training.\n",
```

Comment:
```suggestion
    "One helpful advantage of having access to the training loop is that you can now use your own DataLoaders during training of the density estimator. In this fashion, larger datasets can be used as input to `sbi` where `x` is potentially an image or something else. While this will require [embedding the input data](https://sbi-dev.github.io/sbi/latest/tutorials/04_embedding_networks/), a more fine grained control over loading the data is possible and allows to manage the memory requirement during training.\n",
```